### ISSUE TITLE
GLSL 3.30 not supported

### ISSUE THREAD

**Issue by Dradeliomecus** [Dradeliomecus/WorldOfChaos#11]
On the Ubuntu virtual machine (MSI Laptop), this error occurs while the application is loading:

```
CoreEngine has been initialized.
CoreEngine is starting.
It took 130.969564ms to load 1 fonts.
0:1(10): error: GLSL 3.30 is not supported. Supported versions are: 1.10, 1.20, 1.30, 1.40, 1.00 ES, and 3.00 ES

AL lib: (EE) alc_cleanup: 1 device not closed
```

**Comment by Dradeliomecus** [comment]
Solves the crash.

Everything is black screen but game works (button quit works).
This is in Ubuntu 20.04 LTS on VM.

**Comment by Dradeliomecus** [comment]
Fixed with commit 0bc58a6404aa6d4631b46dabf340ea36c08b3ef9.
